gm4_crossbow_cartridges/data/gm4_crossbow_cartridges/predicates/check_block/below.json

```diff
@@ -65,17 +65,6 @@
 				}
 			}
 		},
-		{
-			"condition": "minecraft:location_check",
-			"predicate": {
-				"block": {
-					"blocks": ["minecraft:piston_head"],
-					"state": {
-						"facing": "up"
```

Comment:
Why is it no longer valid to place torches on top of an extended piston?

Reply:
Not sure why that was removed, I think I just forgot to re-add it when testing things